src/battle-scene.ts

```diff
@@ -2773,16 +2782,36 @@ export default class BattleScene extends SceneBase {
     return (player ? this.modifiers : this.enemyModifiers).filter((m): m is T => m instanceof modifierType);
   }
 
+  /**
+   * Get all of the modifiers that pass the `modifierFilter` function
+   * @param modifierFilter The function used to filter a target's modifiers
+   * @param player Whether to search the player (`true`) or the enemy (`false`); Defaults to `true`
+   * @returns the list of all modifiers that passed the `modifierFilter` function
+   */
   findModifiers(modifierFilter: ModifierPredicate, player: boolean = true): PersistentModifier[] {
```

Comment:
same comments to other functions (add types on param docs and player -> isPlayer)
```suggestion
   * @param {ModifierPredicate} modifierFilter The function used to filter a target's modifiers
   * @param isPlayer Whether to search the player (`true`) or the enemy (`false`); Defaults to `true`
   * @returns the list of all modifiers that passed the `modifierFilter` function
   */
  findModifiers(modifierFilter: ModifierPredicate, isPlayer: boolean = true): PersistentModifier[] {
```

Reply:
Can we drop this comment after the discord feedback?

Reply:
resolved 👍